docs/src/main/sphinx/release/release-437.md

```diff
@@ -0,0 +1,37 @@
+# Release 437 (17 Jan 2024)
+
+## General
+
+* Add support for `char(n)` values in {func}`to_utf8`. ({issue}`20158`)
+* Add support for `char(n)` values in {func}`lpad`. ({issue}`16907`)
+* {{breaking}} Replace the `exchange.compression-enabled` configuration property
+  and `exchange_compression` session property with `exchange.compression-codec`
+  and `exchange_compression_codec`, respectively, which allow specifying `NONE`,
```

Comment:
```suggestion
  and `exchange_compression_codec`, respectively.
```

link to docs instead of having all the details here .. let me know if you need an anchor and I can add in https://github.com/trinodb/trino/pull/20458